frontend/src/app/home/home.page.scss

```diff
@@ -24,6 +24,25 @@
 h1,
 h3 {
   text-align: center;
+<<<<<<< HEAD
+}
+
+.fancy-button {
+  font-size: 26px;
+  color: var(--ion-color-secondary-contrast);
+  background: var(--ion-color-secondary);
+  padding: 1rem;
+  border-radius: 1000px 0 1000px 1000px;
+  font-weight: 500;
+  letter-spacing: 4px;
+  width: 100%;
+  outline: none;
+
+  // Add some spacing so that the button is not "pressed" against the above content when the screen has a small height.
```

Comment:
Resolve conflict by using incoming change sets. CSS in this file has been moved.